packages/shared/src/lib/featureManagement.ts

```diff
@@ -39,6 +39,7 @@ const feature = {
   onboardingMostVisited: new Feature('onboarding_most_visited', false),
   shareExperience: new Feature('share_experience', false),
   bookmarkLoops: new Feature('bookmark_loops', false),
```

Comment:
So are we using bookmarkLoops anymore? Looks like we aren't.. in this case, shall we remove or just not add another flag `bookmarkPopup` but keep `bookmarkLoops` and just adjust it's usage?